ctapipe/image/extractor.py

```diff
@@ -935,7 +925,7 @@ def _apply_second_pass(
         # We have to add 2 samples each side, so the shift will always
         # be (-)2, while width will always end 4 samples to the right.
         # This "always" refers to a 5-samples window of course
-        window_width_default = 5
+        window_width_default = 5  # +1 since extract_around_peak stops at end
```

Comment:
THis comment and the one above don't really make sense together and I think they can be simplified to "use an integration window of width 5 central around the peak."